### ISSUE TITLE
Vendor Staff Manager discloses the Admin wp-login Page

### ISSUE THREAD

**Issue by Tangol** [weDevsOfficial/dokan#1175]
Dokan [Vendor Staff Manager](https://wedevs.com/docs/dokan/modules/dokan-vendor-staff-manager/admin-settings-vendor-dashboard-staff-login/) module uses the default WP user welcome email template, which sends a confirmation email to the staffer when his account gets created, therefore disclosing the `wp-login` page. 

```
Username: [user_login]
To set your password, visit the following address: [password_url]
```

This can count as a security issue, as any respectable store should hide the admin login page in order to protect from hackers, using plugins like [this](https://wordpress.org/plugins/wps-hide-login/).
Also, it kinda defeats the purpose of having a front-end shop management system (`Disallow Vendors and Customers from accessing the wp-admin dashboard area`), if the users can actually see the admin login page (we keep them in the front, so that we can protect the back).

Bottom line is that the Dokan **Vendor Staff Manager** module should come with its own user password setup page, visually independent of the `wp-admin` one.
Or if this is too hard to implement, then give the Vendor the possibility to generate a password for the Staffer at account creation, using the default WP Password Generator, without sending an email.

![password](https://user-images.githubusercontent.com/3336728/115021101-33840200-9ec4-11eb-8269-ab22d0e4501a.png)

Thanks!
